src/markov/markov_approx.jl

```diff
@@ -219,3 +223,567 @@ function estimate_mc_discrete{T}(X::Vector{T})
 
     return estimate_mc_discrete(X, states)
 end
+
+"""
+
+types specifying the method for `discrete_var`
+
+"""
+@compat abstract type AbstractMethod end
```

Comment:
I think this should be more descriptive than just `AbstractMethod`. Maybe `VAREstimationMethod` (I think if we have something descriptive then we don't need the `Abstract`)

Open to other thoughts here.